time/src/lib.rs

```diff
@@ -300,4 +300,14 @@ impl UTime {
         let datetime: DateTime<Utc> = DateTime::from_utc(naive, Utc);
         format!("{}", datetime)
     }
+
+    pub fn days_hours_mins_secs(&self) -> (u64, u64, u64, u64) {
```

Comment:
I'm no sure about the naming, and maybe it would be more obvious to return a custom named struct rather than a tuple struct?

Reply:
Any ideas on the name of the struct ?

Reply:
why not just return a formatted string ?

pub fn format_human_readable(&self) -> String

you can also use: https://crates.io/crates/hhmmss

Reply:
This crate does not give days.

format human readable is ambiguous : is it human readable as a date or as a duration ?